magicctapipe/scripts/lst1_magic/lst1_magic_stereo_reco.py

```diff
@@ -142,25 +153,36 @@ def stereo_reconstruction(input_file, output_dir, config, magic_only_analysis=Fa
     # Apply the event cuts
     logger.info(f"\nMAGIC-only analysis: {magic_only_analysis}")
 
+    LSTs_IDs = np.asarray(list(assigned_tel_ids.values())[0:4])
+
     if magic_only_analysis:
-        event_data.query("tel_id > 1", inplace=True)
+        event_data.query(f"tel_id > {LSTs_IDs.max()}", inplace=True) # Here we select only the events with the MAGIC tel_ids, i.e. above the maximum tel_id of the LSTs
 
     logger.info(f"\nQuality cuts: {config_stereo['quality_cuts']}")
-    event_data = get_stereo_events(event_data, config_stereo["quality_cuts"])
+    event_data = get_stereo_events(event_data, config=config, quality_cuts=config_stereo["quality_cuts"])
 
-    # Check the angular distance of the LST-1 and MAGIC pointing directions
+    # Check the angular distance of the LST and MAGIC pointing directions
     tel_ids = np.unique(event_data.index.get_level_values("tel_id")).tolist()
 
-    if (not is_simulation) and (tel_ids != [2, 3]):
+    Number_of_LSTs_in_use = len(LSTs_IDs[LSTs_IDs > 0]) 
+    MAGICs_IDs = np.asarray(list(assigned_tel_ids.values())[4:6])
```

Comment:
another dependence of sequence of telescope names in the config file

Reply:
See the previous reply about the check_input_list function. This function will force the user to input the correct list of telescopes and IDs.